kubeflow/pipeline/all.libsonnet

```diff
@@ -0,0 +1,33 @@
+{
+  parts(_env, _params):: {
+    local params = _env + _params,
+
+    local minio = import "kubeflow/pipeline/minio.libsonnet",
+    local mysql = import "kubeflow/pipeline/mysql.libsonnet",
+    local pipeline_apiserver = import "kubeflow/pipeline/pipeline-apiserver.libsonnet",
+    local pipeline_scheduledworkflow = import "kubeflow/pipeline/pipeline-scheduledworkflow.libsonnet",
+    local pipeline_persistenceagent = import "kubeflow/pipeline/pipeline-persistenceagent.libsonnet",
+    local pipeline_ui = import "kubeflow/pipeline/pipeline-ui.libsonnet",
+    local spartakus = import "kubeflow/pipeline/spartakus.libsonnet",
```

Comment:
spartakus is also available here: https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/prototypes/spartakus.jsonnet
Can we merge this two?

Reply:
Will pipeline always be installed? If so maybe we can remove it .

Reply:
The plan is to install pipelines by default.
@abhi-g is looking into how we can provide additional information (e.g. about installed components) with spartakus.

So maybe we can remove spartakus for now and look into how to add pipeline specific data to that.

Reply:
The plan is to install pipelines by default.
@abhi-g is looking into how we can provide additional information (e.g. about installed components) with spartakus.

So maybe we can remove spartakus for now and look into how to add pipeline specific data to that.

Reply:
SG. Removed it.